tests/TestCase/Identifier/LdapIdentifierTest.php

```diff
@@ -82,6 +82,24 @@ public function testIdentify()
     }
 
     /**
+     * testWrongLdapObject
+     *
+     * @return void
+     * @expectedException \RuntimeException
+     * @expectedExceptionMessage Could not build the LDAP connection object.
+     */
+    public function testWrongLdapObject()
+    {
+        $identifier = new LdapIdentifier([
+            'host' => 'ldap.example.com',
+            'bindDN' => function () {
+                return 'dc=example,dc=com';
+            },
+            'ldapClass' => 1
```

Comment:
A string would be handeld by the `is_string` condition, an unknown class would raise an Error and an anonymous class is only available in php 7+